app/views/users/index.html.erb

```diff
@@ -3,8 +3,11 @@
   <tbody>
     <% @users.each do |user| %>
     <tr>
-      <td><%= user.name %></td>
-      <td><%= user.posts_counter %></td>
+      <div class='user_card'>
+        <%= image_tag user.photo%>
+        <h2><%= link_to(user.name, "users/#{user.id}")%></h2>
+        <small>Number of posts:<%= user.posts_counter%></small>
+      </div>
     </tr>
     <% end %>
   </tbody>
```

Comment:
- [OPTIONAL]
- [ ] Instead of using `@object.each`, you can check out the option below as a better practice.
```ruby
  # app/views/users/_user.html.erb

     #   HTML Code Here displaying
     #   user.photo, user.name and user.posts_counter

    # app/views/users/index.html.erb

    <%= render @users %>
```